apiary.geojson

```diff
@@ -0,0 +1,33 @@
+{
+    "type": "FeatureCollection",
+    "features": [{
+        "type": "Feature",
+        "properties": {
+            "id": 1,
+            "name": "Vahekoidu",
+            "address": "Vahekoidu tee 6, Koidu küla, Saue vald, Harju maakond, Estonia",
+            "code": "EE31636",
```

Comment:
This is the code given by authority (PRIA in Estonia). Is it needed to add an extra value to describe that authority?